frontend/src/components/course/components/modules-cards-container/components/module-card/module-card.tsx

```diff
@@ -1,22 +1,48 @@
+import { StringCase, TaskStatus } from 'common/enums/enums';
 import { FC } from 'common/types/types';
 import { Content } from 'components/common/common';
+import { changeStringCase, getValidClasses } from 'helpers/helpers';
 
 import styles from './styles.module.scss';
 
 type Props = {
   title: string;
   description: string | null;
+  status?: TaskStatus;
 };
 
-const ModuleCard: FC<Props> = ({ title, description }) => {
+const ModuleCard: FC<Props> = ({ title, description, status }) => {
+  const camelCaseStatus =
+    status &&
+    changeStringCase({
+      caseType: StringCase.CAMEL_CASE,
+      stringToChange: status,
+    });
+
   return (
-    <div className={styles.card}>
-      <div className={styles.title}>
-        <p className={styles.header}>{title}</p>
-      </div>
-      <div className={styles.description}>
-        {description && <Content html={description} />}
+    <div className={styles.container}>
```

Comment:
now we don't want to render a description at all. only a name and a status

Reply:
We don't render it, if there's status (if it is a page for a mentor)
```
{!status && (
   <div className={styles.description}>
      {description && <Content html={description} />}
   </div>
)}
```
But we should render it for student, shouldn't we?

Reply:
no, we shouldn't. now this content should be only on the module page